todo_app/data/trello_items.py

```diff
@@ -1,102 +1,107 @@
 from todo_app.data.todo_items import Item
```

Comment:
Updating the name of this file now feels like it would be worthwhile, or future developers will be quite confused!